src/auth/internal/AuthIntegrationInterface.ts

```diff
@@ -82,13 +82,16 @@ function sendSms(phoneNum: string, featheryClient: any) {
   }
 }
 
-function verifySms(params: {
-  fieldVal: string;
-  featheryClient: any;
-}): Promise<any> {
+function verifySMSEmailOTP(
```

Comment:
the email OTP verification should call featheryClient.verifyOTP directly instead of going through this interface